packages/pipeline/src/utils/getPipelineCounterTooltip.ts

```diff
@@ -0,0 +1,28 @@
+import { ReactNode } from 'react';
+import { onlyText } from 'react-children-utilities';
+import { flatMap } from 'lodash';
+import { flow, compact, split, map } from 'lodash/fp';
+
+/**
+ * A utility function which takes the React.children rendered by the Pipeline component
+ * and returns the string which represents the full pipeline, for usage with the tooltip.
+ *
+ * @param children - the children rendered by the pipeline component
+ * @returns string - the tooltip text
+ */
+export function getPipelineCounterTooltip(children: ReactNode): string {
+  const stages = flow(
```

Comment:
Perhaps I'm missing something, so let me know if I am. Couldn't this be written as this?
```
const stages = children.map(onlyText)
```

As a note, we do tend to prefer native methods to lodash when possible for bundle size, and readability reasons.

Reply:
Fair enough, I opted to use a functional programming style here, but if that is not aligned with the norm on LeafyGreen I can change.

Reply:
Resolved by ~~fc2aade~~ 52053b1